scripts/apps/authoring/metadata/PlacesService.ts

```diff
@@ -1,13 +1,42 @@
 import {omit} from 'lodash';
 
+interface IGeoname {
+    name: string;
+    code: string;
+
+    state: string;
+    country: string;
+
+    state_code: string;
+    country_code: string;
+
+    tz: string;
```

Comment:
Add comments for unclear fields. These aren't clear to me:

- `IGeoname['name']` - what does it contain? Full adress, coordinates or something else?
- `IGeoname['code']`
- `IGeoname['tz']`
- `IGeoname['scheme']`

- `ILocated['dateline']`
- `ILocated['tz']`
- `ILocated['alt_name']`
- `ILocated['code']`
- `ILocated['scheme']`